tests/themes/oe_theme_subtheme_test/config/schema/oe_theme_subtheme_test.schema.yml

```diff
@@ -0,0 +1,7 @@
+oe_theme_subtheme_test.settings:
+  type: theme_settings
+  label: 'OpenEuropa Theme subtheme test settings'
```

Comment:
On this let's use the following:
````
oe_theme_subtheme_test.settings:
  type: oe_theme.settings
  label: 'OpenEuropa Theme subtheme test settings'
```
This is the right way of inheriting schema from the parent theme. \cc @upchuk